aleemer/Internify
# Feat/in76 profile pulls data

#### User Story
https://chaosneutral.atlassian.net/browse/IN-76

#### Changes made
The profile page now displays user information obtained from the store (which itself obtains data from the database) 
Also, got rid of warning message by changing "justify" to "justifyContent" where applicable
Finally, fixed some stylistic problems regarding the logo and the add job button, as well as the profile picture.

#### Does your new code introduce new warnings on dev console?
No

#### Test Steps
(1) run the site
(2) log in with a pre-existing account, or create an account
(3) when you are directed to the Profile page you will notice that at the very least, the account's email address is correctly displayed
